qiskit/providers/options.py

```diff
@@ -26,10 +26,122 @@ class Options(types.SimpleNamespace):
     options.
     """
 
+    _fields = {}
+
+    def __init__(self, **kwargs):
+        self._fields = {}
+        self._fields.update(kwargs)
+        self.validator = {}
+
+    def __repr__(self):
+        items = (f"{k}={v!r}" for k, v in self._fields.items())
+        return "{}({})".format(type(self).__name__, ", ".join(items))
+
+    def __eq__(self, other):
+        if isinstance(self, Options) and isinstance(other, Options):
+            return self._fields == other._fields
+        return NotImplemented
+
+    def set_validator(self, field, validator_value):
+        """Set an optional validator for a field in the options
+
+        Setting a validator enables
```

Comment:
Missing tail of sentence.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/5885/commits/071e7d80b976656a5b8eb9fe7a3c474b1dafd983